yt/admin/dashboards/yt_dashboards/common/sensors.py

```diff
@@ -13,16 +13,20 @@ class YtSystemTags(Enum):
 
 yt_host = YtSystemTags.HostContainer
 
+MonitoringServiceTag = MonitoringTag("service")
+# TODO(achulkov): This is actually promised to be "job" in README. Should we change it?
```

Comment:
What do you mean in this comment?

Reply:
[Here](https://github.com/ytsaurus/ytsaurus/tree/main/yt/admin/dashboards#yt-helpers) under `ProjectSensor` we say that it is `service=` for Monitoring and `job=` for Grafana.

Reply:
Our testing installation had `job=`, that's why I included it into the readme. I encourage you changing readme as well if you change the name here.

Reply:
Frankly, this differs too much depending on your specific Grafana installation + collection specifics. I guess I will change to `service` in both places and add some more details in the readme on what this label should represent.